ProcessingShowPath/Path.pde

```diff
@@ -0,0 +1,12 @@
+class Path{ 
+  String[][] gameplayData;    //coordinates and tags from the file
+  String participantNumber;   
+  int pathLength;             //length of gameplayData, used when drawing single file
+  
+  Path(String[][] gameplayData, String participantNumber, int pathLength)
+  {
+    this.gameplayData = gameplayData;
+    this.participantNumber = split(participantNumber, "/")[1];
+    this.pathLength = pathLength;
+  }
```

Comment:
From what I found, `pathLenght` is always just `gamePlayData.length`. Can this `pathLength` variable be set in the constructor instead of passing it as an argument?